### PR TITLE
MueLu: Adding Pairwise to easy parameterlist

### PR DESCRIPTION
**Pull request by csiefer2**
Tests: RHEL6

### CODE REVIEW

File: packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp [hunk 1]
```diff
@@ -971,7 +972,7 @@ namespace MueLu {
 
     // Aggregation scheme
     MUELU_SET_VAR_2LIST(paramList, defaultList, "aggregation: type", std::string, aggType);
-    TEUCHOS_TEST_FOR_EXCEPTION(!strings({"uncoupled", "coupled", "brick", "matlab"}).count(aggType),
+    TEUCHOS_TEST_FOR_EXCEPTION(!strings({"uncoupled", "coupled", "brick", "matlab","notay"}).count(aggType),
```
Comment (jhux2) [inline packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp:975]: "pairwise" might be more descriptive, my 2 cents.

Comment (csiefer2) [reply packages/muelu/src/Interface/MueLu_ParameterListInterpreter_def.hpp:975]: "Notay"  equals the algorithm as described in the paper (or as close as we could come)

"Pairwise" (when we have it)  will describe "our fire-breathing custom pairwise algorithm"

**Comment by csiefer2** [comment]
> 
> 
> Seems ok. Can't believe you only need to update two gold files, though.

I don't believe it either :)  But the Oracle of the Autotester will enlighten me!

**Comment by csiefer2** [comment]
Repeatec restarts.  Awesome.  @trilinos/framework

**Pull request merged**